include/wx/graphics.h

```diff
@@ -286,6 +269,201 @@ class WXDLLIMPEXP_CORE wxGraphicsMatrix : public wxGraphicsObject
 
 extern WXDLLIMPEXP_DATA_CORE(wxGraphicsMatrix) wxNullGraphicsMatrix;
 
+// ----------------------------------------------------------------------------
+// wxGradientStop and wxGradientStops: Specify what intermediate colors are used
+// and how they are are spread out in a gradient
+// ----------------------------------------------------------------------------
+
+// Describes a single gradient stop.
+class wxGraphicsGradientStop
+{
+public:
+    wxGraphicsGradientStop(wxColour col = wxTransparentColour,
```

Comment:
Do we really need `wxTransparentColour` here or would just `wxColour()` do?

Reply:
It wouldn't have the same meaning. `wxTransparentColour` is a valid colour, but `wxColour()` is not. Also, this is preexisting code that was just moved to fix compilation errors.